7024Movies/Pages/DiscoverThriller.cshtml.cs

```diff
@@ -28,21 +28,21 @@ public void OnGet()
                 JObject jsonObject = JObject.Parse(JsonString);
                 if (jsonObject.IsValid(schema))
                 {
-                    var discoverthriller = DiscoverM.DiscoverMovie.FromJson(JsonString);
-                    var thrillermovies = discoverthriller.Results;
-                    ViewData["Thrillers"] = thrillermovies;
+                    var DiscoverThriller = DiscoverM.DiscoverMovie.FromJson(JsonString);
+                    var ThrillerMovies = DiscoverThriller.Results;
+                    ViewData["Thrillers"] = ThrillerMovies;
```

Comment:
By this best practices article it talks about using Carmelcase when declaring variables.
https://dev.to/prahladyeri/underscores-camelcasing-and-pascalcasing-the-three-naming-conventions-every-programmer-should-be-aware-of-3aed

Reply:
Java tends to be lowerCamelCase, while C# leans toward UpperCamelCase.  Either way, those are an improvement over alllowercase.